webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue

```diff
@@ -556,8 +556,9 @@ export default {
         this.postNews();
       }
     },
-    postNews: function (schedulePostDate) {
-      if (this.news.pinned === true) {
+    postNews: function (schedulePostDate, postArticleMode, publish) {
+      this.news.pinned = (publish === 'true');
+      if (publish !== 'true' && this.news.pinned === true) {
```

Comment:
It seems that these two conditions are contradictious no ?

Reply:
no,we use it to ensure that we don't show confirm dialog when we publish from drawer.

Reply:
confirm dialog to be removed from news composer since no more used